externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/doris/DorisJdbcOperationManager.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kyuubi.engine.jdbc.doris
+
+import org.apache.kyuubi.config.KyuubiConf.OPERATION_INCREMENTAL_COLLECT
+import org.apache.kyuubi.engine.jdbc.operation.JdbcOperationManager
+import org.apache.kyuubi.operation.Operation
+import org.apache.kyuubi.session.Session
+
+class DorisJdbcOperationManager extends JdbcOperationManager {
```

Comment:
how about making it more common?

JdbcOperationManager shall be able to share across JDBC servers

Reply:
There are some differences between the jdbc sources. 

1. GetXXXOperations have different grammar.
2. The jdbc type to hive type mapping is different, for example, the value of tinyint type is classOf Integer in doris.
3. Setting query fetch size have different way.

The differences leads us to distinguish between different types.

Reply:
I agree the differences，but I do not like the way handle them

Reply:
refer to spark's jdbc datasource, add a jdbc dialect handle the differencts.